actors/miner/tests/prove_commit.rs

```diff
@@ -511,11 +511,18 @@ fn drop_invalid_prove_commit_while_processing_valid_one() {
         verify_deals_exit: HashMap::from([(sector_no_a, ExitCode::USR_ILLEGAL_ARGUMENT)]),
         ..Default::default()
     };
-    h.confirm_sector_proofs_valid(&rt, conf, vec![pre_commit_a, pre_commit_b]).unwrap();
+    h.confirm_sector_proofs_valid_for(
+        &rt,
+        conf,
+        vec![pre_commit_a, pre_commit_b],
+        vec![sector_no_b],
+    )
+    .unwrap();
     let st = h.get_state(&rt);
     assert!(st.get_sector(&rt.store, sector_no_a).unwrap().is_none());
     assert!(st.get_sector(&rt.store, sector_no_b).unwrap().is_some());
     h.check_state(&rt);
+    rt.reset();
```

Comment:
This call didn't abort, so this should `rt.verify()` instead. Or you can skip it because it's automatic when dropped, which happens immediately next. If you added this because expectations are failing, something is wrong.

Reply:
Fixed, we don't need this. It's a remanent of something I was debugging earlier.